apps/payment/models.py

```diff
@@ -362,10 +375,12 @@ def _handle_status_change(self):
             """ Handle the completed payment. Remove delays, suspensions and marks """
             self.payment.handle_payment(self.user)
             self.status = status.DONE
+            self._create_fiken_sale(self.payment_price.price)
         elif self.status == status.REFUNDED:
             self.refunded = True
             self.payment.handle_refund(self)
             self.status = status.REMOVED
+            self._create_fiken_sale(-self.payment_price.price)
```

Comment:
Should this be done before the status is set? Just so that in case something here goes wrong, it's not marked as REMOVED?

Reply:
Yeah, that sounds reasonable

Reply:
But would it not count anyways, since the save method won't be saved if an error occurs?

Reply:
I'm not sure, I didn't look that deep into the existing payment code, but that might be the case.

Reply:
I switched them anyways :man_shrugging: